analysis_notebooks/Geography Analysis.ipynb

```diff
@@ -731,13 +776,52 @@
     "                subregion_pvalues_over[p] = tag_info\n",
     "            else:\n",
     "                subregion_pvalues_under[p] = tag_info\n",
+    "                \n",
+    "elif (dataset.geography_info_type == \"STRING_FORMATTED_LABEL\" and dataset.geography_label_string_type == \"REGION_LABEL\"):\n",
+    "    print(\"Geo_tag work for region label formatted dataset\")\n",
+    "    if not os.path.exists(\"results/{0}/6\".format(folder_name)):\n",
```

Comment:
I think "6" has been deprecated for the new names of the analyses, please map accordingly

Reply:
done, replaced '6' with "geo_tag"